app/assets/stylesheets/active_admin_sidebar_pure_icons.scss

```diff
@@ -0,0 +1,62 @@
+/* .icono-caret-left */
+.icono-caret-right,
+.icono-caret-left {
+  width: 19px;
+  height: 19px;
+}
+.icono-caret-right:before,
+.icono-caret-right:after,
+.icono-caret-left:before,
+.icono-caret-left:after {
+  width: 8px;
```

Comment:
Properties should be ordered bottom, box-shadow, height, margin, position, right, transform-origin, width